cucumber/api/features/support/machine_helpers.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+#
+# Utility methods for performing actions in remote machines
+#
```

Comment:
I kind of feel like `machine**` is maybe too generic. What about `ssh_**` instead or something similar?

Reply:
@sgnn7 i was debating between `ssh_helpers` and `machine_helpers` and went with the latter for 2 reasons:
1. this is what we have in appliance: https://github.com/conjurinc/appliance/blob/master/features/step_definitions/machine_steps.rb
2. we may want to add in future more abilities such as to scp to a machine.

what do you think? go with `ssh_helpers` and add `scp_helpers` in the future or stay with machine?

Reply:
Hmm... I see your reasoning and it makes sense though "machine" is pretty ambiguous even though we have it in the appliance. Since `scp` is done over `ssh`, I think `ssh_helpers` would still be valid but even `remote_machine_helpers` or something similar would be fine as long as it's not just plain `machine_*`.

Reply:
renamed to `ssh_`, thanks!